backend/app/app.auth.js

```diff
@@ -32,8 +32,53 @@ passport.deserializeUser((user, done) => {
 });
 
 const auth = {
-  initialize: () => passport.initialize(),
-  session: () => passport.session(),
+  initialize() {
+    return passport.initialize();
+  },
+
+  session() {
+    return passport.session();
+  },
+
+  isUserLoggedIn(req) {
+    return !!this.currentUser(req);
+  },
+
+  currentUser(req) {
+    return req.user;
+  },
+
+  authenticate(opts) {
```

Comment:
What do you think about moving this method to a file inside a directory of middlewares? Example:

`backend/app/middlewares/authenticate.js`

Is not clear to me that this method is a middleware. I only found out when I saw the usage on the route.

What do you think?